config.js

```diff
@@ -1,7 +1,7 @@
 const breakpoints = {
     desktop: 1040,
     tablet: 840,
```

Comment:
Tablet size should be 768 px! Please correct this.

Reply:
Fixed

Reply:
Good